packages/client/src/components/TopCastDisplay/TopCastDisplay.jsx

```diff
@@ -81,49 +81,56 @@ export const TopCastDisplay = ({ movieID }) => {
   return (
     <MovieDetailsLayout>
       <Title>TOP CAST</Title>
-      <Stack
-        direction="row"
-        spacing={1}
-        sx={{ display: 'flex', justifyContent: 'center', flexWrap: 'wrap' }}
-      >
-        {castList
-          .slice(0, showFullCast ? castList.length : 6)
-          .map((castMember) => (
-            <Box
-              key={castMember.id}
-              sx={{
-                width: '13.75rem',
-                p: 1,
-                textAlign: 'center',
-                display: 'flex',
-                flexDirection: 'column',
-                justifyContent: 'space-between  ',
-                alignItems: 'center',
-              }}
-            >
-              <StyledAvatar
-                alt={`${castMember.full_name}`}
-                src={castMember.image_location}
-                variant="circular"
-              />
-              <StyledCastTitle variant="caption">
-                {`${castMember.full_name}`}
-              </StyledCastTitle>
-            </Box>
-          ))}
-      </Stack>
-      {isLoading ? (
+      {castList.length === 0 ? (
+        <StyledTypography>
+          No cast members available for this movie
+        </StyledTypography>
+      ) : (
+        <>
+          <Stack
+            direction="row"
+            spacing={1}
+            sx={{ display: 'flex', justifyContent: 'center', flexWrap: 'wrap' }}
+          >
+            {castList
+              .slice(0, showFullCast ? castList.length : 6)
+              .map((castMember) => (
+                <Box
+                  key={castMember.id}
+                  sx={{
+                    width: '13.75rem',
+                    p: 1,
+                    textAlign: 'center',
+                    display: 'flex',
+                    flexDirection: 'column',
+                    justifyContent: 'space-between  ',
+                    alignItems: 'center',
+                  }}
+                >
+                  <StyledAvatar
+                    alt={`${castMember.full_name}`}
```

Comment:
You could put `castMember.full_name` directly here without the dollar sign and quote marks.